packit_service/worker/parser.py

```diff
@@ -661,17 +661,25 @@ def parse_check_rerun_event(
             return None
 
         check_name_job, check_name_target = None, None
-        # e.g. packit/rpm-build-fedora-34-x86_64
-        check_name_split = check_name.split("/", maxsplit=2)
-        if len(check_name_split) == 2:
-            check_name_job_info = check_name_split[1]
+        if ":" in check_name:
+            # e.g. "rpm-build:fedora-34-x86_64"
+            check_name_job, _, check_name_target = check_name.partition(":")
+            if check_name_job not in MAP_CHECK_PREFIX_TO_HANDLER:
+                logger.warning(
+                    f"{check_name_job} not in {list(MAP_CHECK_PREFIX_TO_HANDLER.keys())}"
+                )
+                check_name_job = None
+        elif "/" in check_name:
```

Comment:
I think you could add `TODO` for removal after some time, I doubt it makes sense to keep this in for longer than a month